pkg/controllers/clusterset/clustersetmapper/clusterset_controller.go

```diff
@@ -221,15 +217,23 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	//add clusterset clusterrole
 	adminroleName := utils.GenerateClustersetClusterroleName(clusterset.Name, "admin")
-	adminRole := buildAdminRole(clusterset.Name, adminroleName)
+	adminRole := clustersetutils.BuildAdminRole(clusterset.Name, adminroleName)
 	err = utils.ApplyClusterRole(r.kubeClient, adminRole)
 	if err != nil {
 		klog.Warningf("will reconcile since failed to create/update clusterrole %v, %v", clusterset.Name, err)
 		return ctrl.Result{}, err
 	}
 
+	bindroleName := utils.GenerateClustersetClusterroleName(clusterset.Name, "bind")
+	bindRole := clustersetutils.BuildBindRole(clusterset.Name, bindroleName)
+	err = utils.ApplyClusterRole(r.kubeClient, bindRole)
```

Comment:
I think we should apply all clusterroles, and return the aggregate err

Reply:
done